main/acle.md

```diff
@@ -2699,11 +2704,26 @@ following:
   in one of the translation units.
   * Implicitly, without this attribute,
   * or explicitly providing the `default` in the attribute.
-* All instances of the versions shall share the same function
-  signature and calling convention.
+* The default version signature is the signature for calling
+  the multiversioned functions.
+* Non-default versions shall share the same calling convention
+  of the default version and have a type that is convertible to the type
+  of the default version.
+
 * All the function versions must be declared at the translation
   unit in which the definition of the default version resides.
 
+For example, the below is valid and 2 is passed as the
+value for `c` when calling `f`.
+
+```C
+int __attribute__((target_version("simd"))) f (int c = 1);
+int __attribute__((target_version("default"))) f (int c = 2);
+int __attribute__((target_version("sve"))) f (int c = 3);
+
+int g() { return f(); }
+```
```

Comment:
Just a suggestion, but it might also be worth adding an example to say that:
```
int f (int c = 1);
int __attribute__((target_version("default"))) f (int c = 2);
```
is not valid, because the two statements are taken to be declaring the same entity.

Reply:
Thank you for the suggestions, Ive added this.